pkg/colorful-hebrew.man

```diff
@@ -0,0 +1,31 @@
+.\" דף הוראות של Colorful
+.\" צרו קשר עם veg@svgames.pl לתיקוני בעיות או שגיאות.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
+.SH NAME
+colorful - משחק יריות פשוט בדו-מימד.
+.SH SYNOPSIS
+\fBcolorful\fR
+.SH DESCRIPTION
+Colorful הוא משחק פשוט יריות פשוט דו-מימדי, בו הפרוטגוניסט מטייל במבוך של 
+מערות ומסדרונות במטרה לאסוף יצירות צבע.
+.SH CONTROLS
+המשחק משוחק בעזרת המקלדת בלבד. הגדרות החצים במתכונת ברירת מחדל:
+.TP
+\fBArrow keys\fR
```

Comment:
"Arrow keys" should also be translated.

Reply:
Alright. Gimme a couple of minutes for that.